### PR TITLE
PROD-628 install jattach

### PR DESCRIPTION
**Pull request by Qi77Qi**
\<your comments for this PR go here\>

---
Have you read [CONTRIBUTING.md](https://github.com/DataBiosphere/leonardo/blob/develop/CONTRIBUTING.md) lately? If not, do that first.

I, the developer opening this PR, do solemnly pinky swear that:

- [ ] I've documented my API changes in Swagger

In all cases:

- [ ] Get a thumbsworth of review and PO signoff if necessary
- [ ] Verify all tests go green (you can re-run automation tests with a comment saying `jenkins retest`
- [ ] Run the automation tests multiple times in parallel to weed out instability if applicable via a comment saying `multi-test`
- [ ] Squash and merge; Delete your branch after this
- [ ] Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

**Review by xuscrbw — APPROVED** [review]

**Comment by rtitle** [comment]
I'm not sure we should be moving away from the AppSec blessed images: https://github.com/broadinstitute/dsp-appsec-blessed-images

**Comment by gpcarr** [comment]
> https://github.com/broadinstitute/dsp-appsec-blessed-images

@rtitle i don't see a blessed image that has jdk instead of jre (since we want jstack). do you know if any blessed image comes with jdk and jstack? if not then should i reach out to appsec asking for one to be added?

**Comment by rtitle** [comment]
Looks like they are all JREs. FWIW you can take a thread dump with `kill -3 <pid>` on a JRE instead of jstack. Heap dump may require JDK though.

I think AppSec wanted to keep the base images as minimal as possible runtime, so JREs. Worth asking whether we could have a blessed JDK image, I think that makes sense for troubleshooting production issues. Another idea is to keep the base image as is, but install something like https://github.com/apangin/jattach or even a full JDK in the Leo Dockerfile (just to get access to JDK tools).

**Comment by rtitle** [comment]
In fact if you can SSH into a Leo pod you could try just installing https://github.com/apangin/jattach manually

**Comment by gpcarr** [comment]
> https://github.com/apangin/jattach

I tried installing what you suggested in case the team needs to use JDK tools in the future!

**Review by xuscrbw — APPROVED** [review]

File: Dockerfile [hunk 1]
```diff
@@ -64,3 +64,7 @@ RUN cd /leonardo && \
 
 # Add Leonardo as a service (it will start when the container starts)
 CMD java $JAVA_OPTS -jar $(find /leonardo -name 'leonardo*.jar')
+
+# Install https://github.com/apangin/jattach to get access to JDK tools
```
Comment (rtitle) [inline Dockerfile]: Nits:

* I'd put this before `CMD`, usually CMD is last
* Does `apt-get` work? Think that's slightly preferred in Dockerfiles

**Review by rtitle — APPROVED** [review]
Couple nits but seems like it works -- if tests pass :shipit:

**Pull request merged**